### PR TITLE
qemu.tests.cfg: Add some scenarios to cover more command in guest

### PR DESCRIPTION
**Pull request by spcui**
Add some scenarios to get event "RESET" and "SHUTDOWN" when
send command in guest.

Signed-off-by: Shuping Cui <scui@redhat.com>

### CODE REVIEW

**Comment by spcui** [comment]
ID: 1143902 1143904

**Comment by spcui** [comment]
# ../../tools/scan_results.py 
Test                                                                                                                                                                                       Status     Seconds  Info
----                                                                                                                                                                                       ------     -------  ----
        (Result file: /root/staf-kvm-devel/autotest-devel/client/results/default/status)
virt.qemu.smp_4.4096m.repeat1.run_test.Host_RHEL.7.1.qcow2.virtio_blk.up.virtio_net.RHEL.6.6.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_quit.from_qmp                       GOOD       50       completed successfully
virt.qemu.smp_4.4096m.repeat1.run_test.Host_RHEL.7.1.qcow2.virtio_blk.up.virtio_net.RHEL.6.6.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_system_reset.from_guest             GOOD       82       completed successfully
virt.qemu.smp_4.4096m.repeat1.run_test.Host_RHEL.7.1.qcow2.virtio_blk.up.virtio_net.RHEL.6.6.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_system_reset.from_qmp               GOOD       43       completed successfully
virt.qemu.smp_4.4096m.repeat1.run_test.Host_RHEL.7.1.qcow2.virtio_blk.up.virtio_net.RHEL.6.6.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_system_powerdown.from_guest         GOOD       28       completed successfully
virt.qemu.smp_4.4096m.repeat1.run_test.Host_RHEL.7.1.qcow2.virtio_blk.up.virtio_net.RHEL.6.6.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_system_powerdown.from_qmp           GOOD       50       completed successfully
virt.qemu.smp_4.4096m.repeat1.run_test.Host_RHEL.7.1.qcow2.virtio_blk.up.virtio_net.RHEL.6.6.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_stop.from_hmp                       GOOD       52       completed successfully
virt.qemu.smp_4.4096m.repeat1.run_test.Host_RHEL.7.1.qcow2.virtio_blk.up.virtio_net.RHEL.6.6.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_resume.from_hmp                     GOOD       18       completed successfully
virt.qemu.smp_4.4096m.repeat1.run_test.Host_RHEL.7.1.qcow2.virtio_blk.up.virtio_net.RHEL.6.6.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_rtc_change.from_guest               GOOD       20       completed successfully
virt.qemu.smp_4.4096m.repeat1.run_test.Host_RHEL.7.1.qcow2.virtio_blk.up.virtio_net.RHEL.6.6.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_watchdog.qmp_pause.from_guest       GOOD       126      completed successfully
virt.qemu.smp_4.4096m.repeat1.run_test.Host_RHEL.7.1.qcow2.virtio_blk.up.virtio_net.RHEL.6.6.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_watchdog.qmp_reset.from_guest       GOOD       113      completed successfully
virt.qemu.smp_4.4096m.repeat1.run_test.Host_RHEL.7.1.qcow2.virtio_blk.up.virtio_net.RHEL.6.6.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_watchdog.qmp_poweroff.from_guest    GOOD       96       completed successfully
virt.qemu.smp_4.4096m.repeat1.run_test.Host_RHEL.7.1.qcow2.virtio_blk.up.virtio_net.RHEL.6.6.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_watchdog.qmp_shutdown.from_guest    GOOD       86       completed successfully
virt.qemu.smp_4.4096m.repeat1.run_test.Host_RHEL.7.1.qcow2.virtio_blk.up.virtio_net.RHEL.6.6.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_watchdog.qmp_debug.from_guest       GOOD       118      completed successfully
virt.qemu.smp_4.4096m.repeat1.run_test.Host_RHEL.7.1.qcow2.virtio_blk.up.virtio_net.RHEL.6.6.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_watchdog.qmp_none.from_guest        GOOD       98       completed successfully
----                                                                                                                                                                                       GOOD       1021

**Comment by vipmike007** [comment]
review failed due to following reason
1. only parse 6 test cases.
2.Some test failed .
                                                                                                                                                                         Status     Seconds  Info
----                                                                                                                                                                                 ------     -------  ----
        (Result file: /root/autotest/test/autotest-devel/client/results/default/status)
virt.qemu.smp_2.8192m.repeat1.run_test.Host_RHEL.7.0.qcow2.virtio_scsi.up.virtio_net.Win8.x86_64.1.io-github-autotest-qemu.qmp_event_notification.qmp_quit.from_qmp                  GOOD       95       completed successfully
virt.qemu.smp_2.8192m.repeat1.run_test.Host_RHEL.7.0.qcow2.virtio_scsi.up.virtio_net.Win8.x86_64.1.io-github-autotest-qemu.qmp_event_notification.qmp_system_reset.from_guest        FAIL       392      Did not receive qmp RESET event notification
virt.qemu.smp_2.8192m.repeat1.run_test.Host_RHEL.7.0.qcow2.virtio_scsi.up.virtio_net.Win8.x86_64.1.io-github-autotest-qemu.qmp_event_notification.qmp_system_reset.from_qmp          GOOD       50       completed successfully
virt.qemu.smp_2.8192m.repeat1.run_test.Host_RHEL.7.0.qcow2.virtio_scsi.up.virtio_net.Win8.x86_64.1.io-github-autotest-qemu.qmp_event_notification.qmp_system_powerdown.from_guest    FAIL       369      Did not receive qmp SHUTDOWN event notification
virt.qemu.smp_2.8192m.repeat1.run_test.Host_RHEL.7.0.qcow2.virtio_scsi.up.virtio_net.Win8.x86_64.1.io-github-autotest-qemu.qmp_event_notification.qmp_system_powerdown.from_qmp      GOOD       38       completed successfully
virt.qemu.smp_2.8192m.repeat1.run_test.Host_RHEL.7.0.qcow2.virtio_scsi.up.virtio_net.Win8.x86_64.1.io-github-autotest-qemu.qmp_event_notification.qmp_stop.from_hmp                  GOOD       38       completed successfully
virt.qemu.smp_2.8192m.repeat1.run_test.Host_RHEL.7.0.qcow2.virtio_scsi.up.virtio_net.Win8.x86_64.1.io-github-autotest-qemu.qmp_event_notification.qmp_resume.from_hmp                GOOD       16       completed successfully
----                                                                                                                                                                                 GOOD       1012

**Comment by spcui** [comment]
hi @vipmike007 
Thanks your comments, already updated some cmd for windows guest.

**Comment by vipmike007** [comment]
win8.1 passed as well

Ack'ed-By: Mike Cao \<bcao@redhat.com\>

**Comment by vipmike007** [comment]
(Result file: /root/autotest/test/staf-kvm-devel/kvm-test/results/default/status)
virt.qemu.smp_2.8192m.repeat1.run_test.Host_RHEL.7.0.qcow2.virtio_scsi.up.virtio_net.Win8.x86_64.1.io-github-autotest-qemu.qmp_event_notification.qmp_quit.from_qmp                  GOOD       40       completed successfully
virt.qemu.smp_2.8192m.repeat1.run_test.Host_RHEL.7.0.qcow2.virtio_scsi.up.virtio_net.Win8.x86_64.1.io-github-autotest-qemu.qmp_event_notification.qmp_system_reset.from_guest        GOOD       58       completed successfully
virt.qemu.smp_2.8192m.repeat1.run_test.Host_RHEL.7.0.qcow2.virtio_scsi.up.virtio_net.Win8.x86_64.1.io-github-autotest-qemu.qmp_event_notification.qmp_system_reset.from_qmp          GOOD       32       completed successfully
virt.qemu.smp_2.8192m.repeat1.run_test.Host_RHEL.7.0.qcow2.virtio_scsi.up.virtio_net.Win8.x86_64.1.io-github-autotest-qemu.qmp_event_notification.qmp_system_powerdown.from_guest    GOOD       20       completed successfully
virt.qemu.smp_2.8192m.repeat1.run_test.Host_RHEL.7.0.qcow2.virtio_scsi.up.virtio_net.Win8.x86_64.1.io-github-autotest-qemu.qmp_event_notification.qmp_system_powerdown.from_qmp      GOOD       40       completed successfully
virt.qemu.smp_2.8192m.repeat1.run_test.Host_RHEL.7.0.qcow2.virtio_scsi.up.virtio_net.Win8.x86_64.1.io-github-autotest-qemu.qmp_event_notification.qmp_stop.from_hmp                  GOOD       40       completed successfully
virt.qemu.smp_2.8192m.repeat1.run_test.Host_RHEL.7.0.qcow2.virtio_scsi.up.virtio_net.Win8.x86_64.1.io-github-autotest-qemu.qmp_event_notification.qmp_resume.from_hmp                GOOD       16       completed successfully

File: qemu/tests/cfg/qmp_event_notification.cfg [hunk 1]
```diff
@@ -10,37 +10,51 @@
...
+                event_cmd = system_reset
         - qmp_system_powerdown:
-            event_cmd = system_powerdown
-            event_cmd_type = qmp_cmd
-            event_check = "POWERDOWN"
+            from_guest:
+                event_check = "SHUTDOWN"
+                event_cmd = shutdown -h now
+                Windows:
+                    event_cmd = shutdown /s /f /t 0
```
Comment (suqinhuang) [inline qemu/tests/cfg/qmp_event_notification.cfg]: so pls put Windows cmd to guest-os.

if you have better idea for the format of configure file, pls open a topic to discuss it.

**Comment by spcui** [comment]
Hi @suqinhuang 
updated with your comments, could you help review again?

**Comment by suqinhuang** [comment]
Acked-by: Suqin Huang

**Comment by spcui** [comment]
https://github.com/avocado-framework/avocado-vt/pull/128

**Comment by ypu** [comment]
As related pullreq already merged so, merge this one to make it work.

Thanks @spcui and @suqinhuang

**Pull request merged**